src/ebay-listbox-button/listbox-button.tsx

```diff
@@ -235,7 +244,7 @@ const ListboxButton: FC<EbayListboxButtonProps> = ({
                     <div
                         className="listbox-button__options"
                         role="listbox"
-                        tabIndex={0}
+                        tabIndex={listContainerTabIndex}
```

Comment:
```suggestion
                        tabIndex={expanded ? 0 : -1}
```
and you don't need any more changes from above